core/arch/x86_64/mm/core_mmu.c

```diff
@@ -1462,10 +1472,8 @@ static void update_pml4_entry(vaddr_t vaddr, uintptr_t pml4_addr, map_addr_t m,
 
 	pml4_table[pml4_index] = m | flags;
 
-	DMSG("pml4_table %p pml4_index 0x%x m 0x%lx flags 0x%lx\n",
```

Comment:
Please restore this log print. You can add pml4_table[pml4_index] print to it if you want.